Lavanya/HTMLassignment3.html

```diff
@@ -0,0 +1,39 @@
+<html>
+<head>
+	<title>HTMLassignment3</title>
+	<meta charset="UTF-8">
+</head>
+<body>
+
+	<h2>Register</h2>
+
+	<form id="RegistrationForm" autocomplete="on">

```

Comment:
`autocomplete` is not required in this place. Individual form entities are good enough.